editors/code/tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"module": "Node16",
+		"target": "ES2022",
+		"lib": [
+			"ES2022",
+			"WebWorker"
+		],
+		"sourceMap": true,
+		"rootDir": "src",
+		"strict": true /* enable all strict type-checking options */
+		/* Additional Checks */
+		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
+		// "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
+		// "noUnusedParameters": true,  /* Report errors on unused parameters. */
```

nit: I think we can turn these on.

```suggestion
		"noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
		"noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
		"noUnusedParameters": true,  /* Report errors on unused parameters. */
```